src/partition.h

```diff
@@ -73,7 +73,8 @@ class SYSTEMSETTINGS_EXPORT Partition
         Unlocking,
         Unlocked,
         Locking,
-        Locked
+        Locked,
+        Renamed
```

Comment:
Is this needed? The mounted/unmounted type of statuses make sense and so do transient mounting/unmounting etc, but this sort of of "end states" feel funny. The "Formatted" is an existing thing, but are they really different from "Unmounted" other than that the partition tracking has seen or initiated some related operation?

Reply:
Done. Removed Renamed status